src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputSimple.java

```diff
@@ -0,0 +1,225 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: Feb-2001
+// Ignore error
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.sizes.linelength;
+
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class InputSimpleLineLength
```

Comment:
input should be named after it's test.
Test is `LineLengthCheckTest`, input should be named `InputLineLength`....
Please rename class and file.
Please do the same for all other inputs in this test.